org.nongnu.Xlog.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: org.nongnu.Xlog
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: xlog
+finish-args:
+  - --device=all
+  - --share=network
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --persist=.xlog
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /doc
+  - '*.a'
+  - '*.la'
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/libusb/libusb.json
+
+  - name: hamlib
+    sources:
+      - type: archive
+        url: https://github.com/Hamlib/Hamlib/releases/download/4.5/hamlib-4.5.tar.gz
+        sha256: 14a2ce76a647a75e71d8ccf1f16999a6708e84823d9f74137f2173fcee94ad52
+        x-checker-data:
+          type: anitya
+          project-id: 1292
+          url-template: https://github.com/Hamlib/Hamlib/releases/download/$version/hamlib-$version.tar.gz
+
+  - name: xlog
+    sources:
+      - type: archive
+        url: https://download.savannah.nongnu.org/releases/xlog/xlog-2.0.24.tar.gz
+        sha256: 8d453ac6ddc7f5b63d0804114c5423a6b96c0bbed5c23201ffab1244dcc4f8bc
+        x-checker-data:
+          type: anitya
+          project-id: 15625
+          url-template: https://download.savannah.nongnu.org/releases/xlog/xlog-$version.tar.gz
+      - type: file
+        path: org.nongnu.Xlog.metainfo.xml
+    post-install:
+      - desktop-file-edit --set-key=Icon --set-value=org.nongnu.Xlog /app/share/applications/xlog.desktop
+      - mv /app/share/applications/xlog.desktop /app/share/applications/org.nongnu.Xlog.desktop
```

Comment:
you should use rename-desktop-file instead.

Reply:
This looks like a more clean solution to me than `rename-desktop-file` since I am already editing the file in post-install.

Reply:
That's your opinion. :-) I am also not sure if `rename-desktop-file` renames the file before or after `post-install` is done - this would need additional testing.

Reply:
after, it's done after everything is built. it also automatically fix the appstream file if needed.

Reply:
Ok, thanks for the info! I have replaced the `mv` command with `rename-desktop-file`. :-)